packages/lib/createAProfileForAnExistingUser.ts

```diff
@@ -17,7 +17,7 @@ export const createAProfileForAnExistingUser = async ({
   user: {
     email: string;
     id: number;
-    username: string | null;
+    currentUsername: string | null;
```

Comment:
Renamed to clearly convey that it is the current username of the user whose org profile is being created.